website/docs/components/reveal/partials/code/component-api.md

```diff
@@ -9,6 +9,9 @@
   <C.Property @name="textWhenOpen" @type="string">
     Plain text which displays on the toggle button while the content is displayed.
   </C.Property>
+  <C.Property @name="ariaDescribedBy" @type="string">
```

Comment:
[todo] usually the documentation is updated in a separate PR, so that it can be merged only when the changes are released; in this case they're minimal, so I'll leave it to you to decide if you want to split the PR or not